.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.5.1
```

Comment:
You are probably fine just specifying 2.5 instead of 2.5.1, however I will not object to being more explicit if that's what you would prefer. :)

Reply:
it's automatically generated so i won't change it if you don't object :)